allensdk/brain_observatory/gaze_mapping/__main__.py

```diff
@@ -35,13 +44,13 @@ def repackage_input_args(parser_args: dict) -> dict:
     """
     new_args: dict = {}
 
-    new_args["input_file"] = Path(parser_args["input_file"])
-    new_args["session_sync_file"] = Path(parser_args["session_sync_file"])
-    new_args["output_file"] = Path(parser_args["output_file"])
+    new_args["input_file"] = parser_args["input_file"]
+    new_args["session_sync_file"] = parser_args["session_sync_file"]
+    new_args["output_file"] = parser_args["output_file"]
 
     monitor_position = np.array([parser_args["monitor_position_x_mm"],
                                  parser_args["monitor_position_y_mm"],
-                                 parser_args["monitor_position_z_mm"]])
+                                 parser_args["monitor_position_z_mm"]]) / 10
```

Comment:
Why is this the only one divided by 10?

Reply:
it's dividing all 3 by 10

```
obtained = np.array([5, 50, 500]) / 10

assert np.allclose(obtained, np.array([0.5, 5, 50]))
```

Reply:
Ahh, I see. I just didn't parse it correctly by eye. No, I think it's fine.